sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -20,6 +20,7 @@ namespace Azure.AI.FormRecognizer.DocumentAnalysis.Tests
     /// These tests have a dependency on live Azure services and may incur costs for the associated
     /// Azure subscription.
     /// </remarks>
+    [IgnoreServiceError(400, "InvalidRequest", Message = "Content is not accessible: Invalid data URL", Reason = "https://github.com/Azure/azure-sdk-for-net/issues/28923")]
```

Comment:
The `IgnoreServiceError` attribute will make sure we don't fail when we encounter these types of errors. **The test won't be retried**, so this is still not ideal. I created GitHub issues (linked in the `Reason` properties) to keep track of removing these attributes once the service has fixed the issue on their side.